### ISSUE TITLE
Godot's language server report functions as... interfaces????

### ISSUE THREAD

**Issue by atirut-w** [godotengine/godot#50815]
### Godot version

3.3.2 stable

### System information

OpenSUSE Tumbleweed(other stuff deemed unnecessary)

### Issue description

I was making a custom vscode extension for Godot stuff and then noticed that functions show up as interfaces. I hopped over to `godotengine/godot-vscode-plugin` and they're also reported as interfaces as well.

![Screenshot_20210725_014919](https://user-images.githubusercontent.com/25323231/126878460-824c747e-7158-44d0-bce6-740c2b791940.png)

And this is not just icons because VSCode explicitly say they're interfaces if I hover my cursor on them in the outline panel.

### Steps to reproduce

Basically using Godot's language server.

### Minimal reproduction project

[TestScript.gd.zip](https://github.com/godotengine/godot/files/6872936/TestScript.gd.zip)

**Comment by Calinou** [comment]
cc @Razoric480

**Comment by Razoric480** [comment]
Looks like the SymbolKind constants were put in wrong, or got updated between when the LSP was written and updated. Will write up a fix.
